src/CallFire-csharp-sdk/Common/Resource/CfSendCall.cs

```diff
@@ -9,13 +9,14 @@ public CfSendCall()
         }
 
         public CfSendCall(string requestId, CfBroadcastType type, string broadcastName, CfToNumber[] toNumber, 
```

Comment:
please do not change interfaces when SDK in live, if users update to this version their code breaks, add another constructor with label, leave this one unchanged